app/models/pusher.rb

```diff
@@ -45,12 +45,12 @@ def save
 
   def pull_spec
     @spec = Gem::Package.new(body).spec
-  rescue StandardError => error
+  rescue StandardError => e
```

Comment:
rubocop complains about this?

Reply:
Yea, this is https://www.rubocop.org/en/latest/cops_naming/#namingrescuedexceptionsvariablename

All the other `rescue`s in `pusher.rb` are using `e` as the variable name - so i thought it good to keep consistency.